crates/dbs-arch/src/lib.rs

```diff
@@ -22,3 +22,45 @@ pub use x86_64::*;
 mod aarch64;
 #[cfg(target_arch = "aarch64")]
 pub use aarch64::*;
+
+/// Enum indicating vpmu feature level
+#[derive(Debug, Eq, PartialEq, Copy, Clone)]
+pub enum VpmuFeatureLevel {
+    /// Disabled means vpmu feature is off (by default)
+    Disabled,
+    /// LimitedlyEnabled means minimal vpmu counters are supported( only cycles and instructions )
+    LimitedlyEnabled,
```

Comment:
If limitedlyEnabled is not used in aarch64, please add a comment here.

Reply:
I have added some comments. Thanks, @studychao